src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/model/mapper/EnvelopeMapperTest.java

```diff
@@ -79,10 +118,34 @@ private void assertSameScannableItems(Envelope dbEnvelope, InputEnvelope zipEnve
 
         assertThat(dbEnvelope.getScannableItems())
             .extracting(this::convertToInputScannableItem)
-            .usingRecursiveFieldByFieldElementComparator()
+            .usingElementComparatorIgnoringFields("ocrData", "documentType", "documentSubtype")
```

Comment:
is it not this already comparing ocr doctype and subtype then do we need this extra test `should_map_document_types_and_subtypes_correctly` ?

Reply:
Next test is focused on comparing types, subtypes and ocr data